modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/views/table/Table.tsx

```diff
@@ -393,18 +394,29 @@ function ClayTableRowOptionalDropTarget({
 	item: any;
 	onItemSelectionChange: Function;
 }) {
+	const [viewsContext] = useContext(ViewsContext);
+
 	const {className: dropClassName, dropRef} = useFDSDrop({item});
 
+	const activeView: IView = viewsContext.activeView;
+
+	const props = {
+		...otherProps,
+		className: classNames(className, dropClassName),
+		items,
+		onClick: () => {
```

Comment:
do we want `setItemComponentProps` to alter this onclick()? Shall there be some stuff we want to keep, like:
 - handlers triggering selection behavior
 - items prop
 - some internal class names, f.e the `drop-target` one

This is a more general question, addressing [your proposal](https://github.com/liferay-frontend/liferay-portal/pull/4997/commits/6841cb12996a56ee3947444e0cbeb849d7fba76c#r2194333846) of applying API last thing.

Being it true to allow overriding, we might consider few exceptions to preserve ootb behavior regardless.